src/applications/gi/utils/helpers.js

```diff
@@ -232,7 +232,27 @@ export const boolYesNo = field => {
   return field ? 'Yes' : 'No';
 };
 
-export const isSmallScreen = () => matchMedia('(max-width: 480px)').matches;
+export const getMobileOperatingSystem = () => {
+  const userAgent = navigator.userAgent || navigator.vendor || window.opera;
+
+  // Windows Phone must come first because its UA also contains "Android"
+  if (/windows phone/i.test(userAgent)) {
+    return 'Windows Phone';
+  }
+
+  if (/android/i.test(userAgent)) {
+    return 'Android';
+  }
+
+  // iOS detection from: http://stackoverflow.com/a/9039885/177710
+  if (/iPad|iPhone|iPod/.test(userAgent) && !window.MSStream) {
+    return 'iOS';
+  }
+
+  return 'unknown';
+};
+
+export const isSmallScreen = () => getMobileOperatingSystem() !== 'unknown';
```

Comment:
Some questions I have: 
1. There are some android and apple devices, mainly tablets, that can be rotated to landscape view. Those would presumably have larger screen sizes. How would that impact the usage of `isSmallScreen` when it's based on User Agent? 
2. There are [some known instances](https://forums.developer.apple.com/forums/thread/119186) where iPad OS uses the same User Agent as Mac OS. Would iPads potentially get missed with the test above?

Reply:
Ipad gets treaded as an IOS device. I'll be running experiments after we deploy. We have a collaboration cycle with Platform Team Tuesday and we'll double check to make sure this is an acceptable solution.